src/webgpu/shader/execution/builtin/abs.spec.ts

```diff
@@ -26,61 +32,48 @@ Component-wise when T is a vector.
   )
   .params(u =>
     u
-      .combineWithParams([{ storageClass: 'storage', storageMode: 'read_write' }] as const)
-      .combine('containerType', ['scalar', 'vector'] as const)
-      .combine('isAtomic', [false])
-      .combine('baseType', ['u32'] as const)
-      .expandWithParams(generateTypes)
+      .combine('storageClass', ['uniform', 'storage_r', 'storage_rw'] as const)
```

Comment:
Future cleanup:  I don't think storage class parameterization is interesting for testing the 'abs' builtin, or any builtin which is a pure function (depending only on its parameters)

Reply:
Mayyyyyybe. HLSL has a `RW` prefix on `ByteAddressBuffer` for mutable buffers. There is some potential for divergence here